src/illuminant/cri.rs

```diff
@@ -191,16 +191,22 @@ mod cri_test {
     #[test]
     #[cfg(feature = "cie-illuminants")]
     fn cri_f1() {
+        use crate::illuminant::CieIlluminant;
         // should be all 100.0
 
         let cri0: CRI = CieIlluminant::F1.illuminant().try_into().unwrap();
         println!("{cri0:?}");
-        // approx::assert_ulps_eq!(cri0.as_ref(), [100.0;crate::cri::N_TCS].as_ref(), epsilon = 0.05);
+        // approx::assert_ulps_eq!(
+        //     cri0.as_ref(),
+        //     [100.0; crate::illuminant::cri::N_TCS].as_ref(),
+        //     epsilon = 0.05
+        // );
```

Comment:
This test doesn't do anything. Because it does not actually check anything. It only prints the CRI. If I uncomment this line, the test fails. However, this PR is not really about making this test work. I just want to make the code compile without allowing the `unused_imports` warning.